x11-drivers/nvidia-drivers/files/bin/setup-nvidia

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+set -euo pipefail
+. /usr/share/flatcar/release
+. /usr/share/flatcar/update.conf
+
+NVIDIA_DRIVER_VERSION=450.80.02
```

Comment:
Since this is hard-coded, new updates won't be installed. Do you think we could fetch the version from somewhere?

Reply:
I've tried to look into this, but there is no official way to fetch the latest versions but agreed that would be most ideal. That said there is an unofficial way where a person maintains the version.

Reply:
The OEM partition contents are not covered by Flatcar updates. We could add a file to /usr where the new version can be found.

Reply:
But anyway, since the installer is not updated we can't react to other changes, too. I guess we have to document how to edit this file manually when people need to update the driver.

Reply:
An ideal scenario would be to bump the ebuild version and the rest would be sorted.